processing/src/main/java/io/druid/query/PrioritizedExecutorService.java

```diff
@@ -50,9 +51,10 @@ public static PrioritizedExecutorService create(Lifecycle lifecycle, ExecutorSer
             config.getNumThreads(),
             0L,
             TimeUnit.MILLISECONDS,
-            new PriorityBlockingQueue<Runnable>(),
+            new PriorityBlockingQueue<Runnable>(config.getNumThreads()),
```

Comment:
removed because it's really not much help to keep it and makes it needlessly complicated